packages/SwingSet/src/vats/vat-timerWrapper.js

```diff
@@ -1,11 +1,13 @@
 import Nat from '@agoric/nat';
 import { assert, details } from '@agoric/assert';
+import { makeNotifierKit } from '@agoric/notifier';
 
 export function buildRootObject(vatPowers) {
   const { D } = vatPowers;
   const repeaters = new Map();
 
   async function createTimerService(timerNode) {
+    /* @type {TimerService} */
```

Comment:
I think this does not work as a JSDoc comment with only one star.
```suggestion
    /** @type {TimerService} */
```

Also, is this line intended to get the types from Zoe? If so, we would need to import those types at the top of this file. But, I think that we do not want SwingSet to have a dependency on Zoe. We should move the `TimerService` type into this package instead and import the TimerService type from the exported.js in this package.

Reply:
Fixed.

TimerService is now declared in SwingSet.